### PR TITLE
Hardcode the export format for the editor 'run' feature

### PR DESCRIPTION
**Pull request by m4gr3d**
The `run` feature generates an `apk` file that is automatically installed on the connected Android device. With the previous logic, switching the export format to Android app bundle (`aab`) for manual export would also negatively impact the `run` feature: the logic would attempt to generate an `aab` file instead of the `apk` file needed by the `run` feature [hence causing an error](https://github.com/godotengine/godot-proposals/issues/342#issuecomment-744988033).

This PR resolves this issue by hardcoding the export format for the `run` feature.

[3.2 backport](https://github.com/godotengine/godot/pull/44639)

### CODE REVIEW

**Review by RandomShaper — APPROVED** [review]

File: platform/android/export/export.cpp [hunk 1]
```diff
@@ -240,6 +240,9 @@ static const LauncherIcon launcher_adaptive_icon_backgrounds[icon_densities_coun
 	{ "res/mipmap/icon_background.png", 432 }
 };
 
+static const int APK_EXPORT_FORMAT = 0;
+static const int AAB_EXPORT_FORMAT = 1;
```
Comment (akien-mga) [inline platform/android/export/export.cpp:244]: Maybe `EXPORT_FORMAT_APK` and `EXPORT_FORMAT_AAB` would be clearer? Otherwise it sounds like those are unrelated constants which define an export format/version for APK and AAB respectively.

Comment (m4gr3d) [reply platform/android/export/export.cpp]: Done!

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!
